test/JDBC/input/BABEL-5119-vu-prepare.mix

```diff
@@ -0,0 +1,27 @@
+-- tsql
+CREATE DATABASE BABEL5119_1
+GO
+
+USE BABEL5119_1
+GO
```

Comment:
Current framework runs on multi-db , right? If yes, then can we please create few more user-created database which are owned by non-sysadmin users and then let's execute has_dbaccess AND some basic permission check. Then only our upgrade changes will be verified.

Reply:
Creating multiple databases resulted in failure of upgrade tests because the upgrade tests are run in single db mode